content/docs/rosa/vpn/_index.md

```diff
@@ -0,0 +1,209 @@
+   set_var EASYRSA_REQ_EMAIL     "me@example.net"
+   set_var EASYRSA_REQ_OU        "My Organizational Unit"
+   ```
+
+   Uncomment (remove the #) the folowing field
+
+   ```
+   #set_var EASYRSA_KEY_SIZE        2048
+   ```
+
+1. Create the CA:
+
+   ```bash
+   ./easyrsa build-ca nopass
+   ```
+
+1. Generate the Server Certificate and Key
+
+   ```bash
+   ./easyrsa build-server-full server nopass
+   ```
+
+1. Generate Diffie-Hellman (DH) parameters
+
+   ```bash
+   ./easyrsa gen-dh
+   ```
+
+1. Generate client credentials
+
+   ```bash
+   ./easyrsa build-client-full aws nopass
+   ```
+
+1. Set environment variables for the CA certificate you just created.
+
+   ```bash
+   CACERT=$(openssl x509 -in pki/ca.crt -outform der | base64)
+   ```
+
+## Import certficates into AWS Certificate Manager
+
+1. Import the server certificate
+
+   * Before running the below commands, make sure you are still in the pki directory under the easyrsa3 directory
+
+    ```bash
+   SERVER_CERT_ARN=$(aws acm import-certificate \
+   --certificate fileb://issued/server.crt \
+   --private-key fileb://private/server.key \ 
+   --certificate-chain fileb://ca.crt \
+   --region $REGION \
+   --query CertificateArn \
+   --output text)
+    ```
+
+1. Import the client certificate
+
+     ```bash
+     CLIENT_CERT_ARN=$(aws acm import-certificate  \ 
+     --certificate fileb://issued/aws.crt \
+     --private-key fileb://private/aws.key \
+     --certificate-chain fileb://ca.crt \
+     --region $REGION \
+     --query CertificateArn \ 
+     --output text)
+    ```
+
+## Create a Client VPN Endpoint
+
+1. Retrieve the VPN Client ID
+
+   ```bash
+    VPN_CLIENT_ID=$(aws ec2 create-client-vpn-endpoint \
+    --client-cidr-block $VPN_CLIENT_CIDR \
+    --server-certificate-arn $SERVER_CERT_ARN \
+    --authentication-options Type=certificate-authentication,MutualAuthentication={ClientRootCertificateChainArn=$CLIENT_CERT_ARN} \
+    --connection-log-options Enabled=false --split-tunnel --query ClientVpnEndpointId --output text)
+   ```
+
+1. Associate each private subnet with the client VPN endpoint
+   
+   ```bash
+   while IFS= read -r subnet;
+   do
+      echo "Associcating subnet '$subnet'"
+      aws ec2 associate-client-vpn-target-network --subnet-id $subnet --client-vpn-endpoint-id $VPN_CLIENT_ID
+   done <<< "$PRIVATE_SUBNET_IDS"
+   ```
+
+1. Add an ingress authorization rule to a Client VPN endpoint
+
+   ```bash
+   aws ec2 authorize-client-vpn-ingress \
+    --client-vpn-endpoint-id $VPN_CLIENT_ID \
+    --target-network-cidr 0.0.0.0/0 \
+    --authorize-all-groups
+   ```
+
+## Configure your OpenVPN Client
+
+1. Download the VPN Client Configuration
+
+   ```bash
+   aws ec2 export-client-vpn-client-configuration --client-vpn-endpoint-id $VPN_CLIENT_ID --output text>client-config.ovpn
+   ```
+
+   * note: make sure you are still in the easy rsa / pki directory.
+
+   ```bash
+   echo '<cert>' >> client-config.ovpn
```

Comment:
oooh I see, this is xml ?

can we do a heredoc ?

```
cat << EOF > client.config.ovpn
<cert>
$(openssl x509 -in issued/aws.crt)
</cert>
<key>
$(cat private/aws.key)
</key>
EOF
```

Reply:
not xml, but 6 different bash commands ... I added a new line between each so it's more readable.